webui/tests/tests/StepImplementation.ts

```diff
@@ -64,6 +65,42 @@ const goto = async (url: string) => {
   await t.goto(url, { waitForEvents: ['loadEventFired'] });
 };
 
+type Eventually<T> = T | Promise<T>;
```

Comment:
sweet! could probably use this in `src/types.ts` wdy think?

Reply:
we already got it there :)

Reply:
in utils/types